src/styles/main.scss

```diff
@@ -1,3 +1,247 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital@1&family=Roboto+Slab:wght@900&family=Roboto:wght@400;500&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital@1&family=Roboto+Slab:wght@900&family=Roboto:wght@400;500;700&display=swap");
+$color-blue: #00acdc;
+$color-grey: #616070;
+$color-black: #060b35;
+$color-white: #fff;
+$padding2pcs: 41px;
+$width-card: 198px;
+$margin3cards: 50px;
+
+@function size($card) {
+  @return 4*$card+3*$margin3cards+2*$padding2pcs;
+}
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.general-container {
+  max-width: size($width-card);
+}
+//**************блок добавленный
+.card {
```

Comment:
and file per each bem block
<img width="653" alt="image" src="https://user-images.githubusercontent.com/102411130/213652716-1b432405-dd46-4f58-8999-5544a215d036.png">